### ISSUE TITLE
URLs with Dashes inside of Layout slots having their url link modified.

### ISSUE THREAD

**Issue by AnAriaEternal** [microsoft-search/pnp-modern-search#2293]
**Version used**
4.7

**Describe the bug**
A block of html is stored within a property on a SharePoint item, that is loaded into the search results via a Refineable String that is entered into a layout slot.  The html going into the slot is pulled in completely, meaning when viewing the html into debug mode, all of the html is full and complete.   However, when that is referenced by **getSummary (slot item @root.slots.RelatedContent)}}** within a custom results template.  Everything is rendering correctly, with the correct formatting and spacing, with one specific exception: URLs that contain four or more dashes have the text next to those first four removed.  Links with three dashes are unaffected, those with five or more have the first four removed but the remainder are unaffected.

Examples:
```
https://URL.sharepoint.com/sites/Site/SitePages/Link-With-Three-Dashes.aspx --> https://URL.sharepoint.com/sites/Site/SitePages/Link-With-Three-Dashes.aspx
https://URL.sharepoint.com/sites/Site/SitePages/Link-With-Four-Dashes-More.aspx --> https://URL.sharepoint.com/sites/Site/SitePages/.aspx 
https://URL.sharepoint.com/sites/Site/SitePages/Link-With-Five-Dashes-Or-More.aspx --> https://URL.sharepoint.com/sites/Site/SitePages/-More.aspx
```

**To Reproduce**
1. Have a layout slot filled with html that contains a href tags.
2. Use **getSummary (slot item @root.slots.RelatedContent)}}** to print those within search results using a custom templte.
3. Load everything onto a page and view the generated links.

**Expected behavior**
The links should all render fully, without url alteration. 

**Screenshots**
![image](https://user-images.githubusercontent.com/111006240/184955915-cf4c0629-7a2d-43a1-a82c-3036f1576a29.png)
![image](https://user-images.githubusercontent.com/111006240/184956178-e91e73d6-2e16-4a02-b952-c755b73a1c5b.png)
![image](https://user-images.githubusercontent.com/111006240/184955667-a3b80883-ded9-4f9e-8f97-eced5b2f34ff.png)
![image](https://user-images.githubusercontent.com/111006240/184955769-9c026fdc-bfcd-44bc-88fa-419e402e0b37.png)

**Desktop (please complete the following information):**
 - Browser: Chrome

**Additional context**
This also applies to other link formats with dashes (such as Microsoft site links.)

**Comment by FranckyC** [comment]
Hi @AnAriaEternal. you don't need to use the `getSummary` helper function. This one is only relevant when you use the default property `HitHighlightedSummary`. To display your HTML just use the triple curly braces syntax like this:

```
{{{slot item @root.slots.RelatedContent}}}
```

**Comment by wobba** [comment]
Fixed with #3042
